galaxykubeman/Chart.yaml

```diff
@@ -2,4 +2,4 @@ apiVersion: v1
 appVersion: 20.01
 description: A chart for deploying a single Galaxy instance and its management dependencies on Kubernetes
 name: galaxykubeman
-version: 0.2.0
+version: 0.2.1
```

Comment:
Should we make 0.2.2 since it seems 0.2.1 already exists in the packaged repo? 
https://github.com/CloudVE/helm-charts/tree/anvil/charts

Reply:
`0.2.1` was a manual packaging with probes staggered in the Galaxy helm chart (i.e. using a temporary fork of galaxy-helm while the fix was made in Galaxy)
I've added some CI https://github.com/galaxyproject/galaxykubeman-helm/pull/6 for versioning and packaging. I'll remove the version bump from the PR and add a label and that will just make it `0.3.0` given that it's a feature.